rdkit/Chem/Fingerprints/FingerprintMols.py

```diff
@@ -326,25 +321,19 @@ def _clusterInit(self):
     self.actName = ''
 
   def GetMetricName(self):
-    if self.metric == DataStructs.TanimotoSimilarity:
-      return 'Tanimoto'
-    elif self.metric == DataStructs.DiceSimilarity:
-      return 'Dice'
-    elif self.metric == DataStructs.CosineSimilarity:
-      return 'Cosine'
-    elif self.metric:
-      return self.metric
-    else:
+    metricDict = {DataStructs.DiceSimilarity: 'Dice', DataStructs.TanimotoSimilarity: 'Tanimoto', 
+                  DataStructs.CosineSimilarity: 'Cosine', } # DataStructs.TverskySimilarity: 'Tversky'
+    try:
+      return metricDict[self.metric]
+    except KeyError:
+      if self.metric:
+        return self.metric
       return 'Unknown'
```

Comment:
How about,
```
Self.metric = metricDict.get(self.metric, self.metric)
If self.metric:
  Return self.metric
Return 'unknown'
```

Reply:
The old code does not re-assign something called `Self.metric`, I would prefer `metric` more

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.